api/src/main/java/org/openmrs/api/impl/ConditionServiceImpl.java

```diff
@@ -48,6 +48,25 @@ public ConditionDAO getConditionDAO() {
 	 */
 	@Override
 	public Condition saveCondition(Condition condition) {
+		Condition existingCondition = getConditionByUuid(condition.getUuid());
```

Comment:
@Arthur236 did you intentionally leave out the first three lines in the original save method of the emrapi module?

Reply:
@dkayiwa yes I did. The Condition class in the core does not have a getter method for the endReason

Reply:
You can add the getter.